src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -321,6 +324,71 @@ def fuzz(self,
                                 artifact_prefix, additional_args, extra_env)
 
 
+class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
+  """libFuzzer runner (when Fuchsia is the target platform)."""
+
+  SSH_RETRIES = 10
+  SSH_WAIT = 2
+
+  def __init__(self, executable_path, default_args=None):
+    if (not environment.get_value('FUCHSIA_PKEY_PATH')) or (
+        not environment.get_value('FUCHSIA_PORTNUM')):
+      raise fuchsia.errors.FuchsiaConfigError(
+          'FUCHSIA_PKEY_PATH and/or FUCHSIA_PORTNUM was not set')
+    # yapf: disable
+    self.ssh_root = [
+        'ssh',
+        '-i', environment.get_value('FUCHSIA_PKEY_PATH'),
+        '-o', 'StrictHostKeyChecking no',
+        '-o', 'UserKnownHostsFile=/dev/null',
+        '-p', str(environment.get_value('FUCHSIA_PORTNUM')),
+        'localhost'
+    ]
+    # yapf: enable
+    super(FuchsiaQemuLibFuzzerRunner, self).__init__(
+        executable_path=executable_path, default_args=default_args)
+
+  def get_command(self, additional_args=None):
+    # TODO(flowerhack): Update this to dynamically pick a result from "fuzz
+    # list" and then run that fuzzer.
+    return self.ssh_command('ls')
+
+  def fuzz(self,
+           corpus_directories,
+           fuzz_timeout,
+           artifact_prefix=None,
+           additional_args=None,
+           extra_env=None):
+    """LibFuzzerCommon.fuzz override."""
+
+    self._test_qemu_ssh()
+    LibFuzzerCommon.fuzz(self, corpus_directories, fuzz_timeout,
+                         artifact_prefix, additional_args)
+
+  def run_single_testcase(self,
+                          testcase_path,
+                          timeout=None,
+                          additional_args=None):
+    #TODO(flowerhack): Fill out this command.
+    pass
+
+  def ssh_command(self, *args):
+    return self.ssh_root + list(args)
+
+  @retry.wrap(retries=SSH_RETRIES, delay=SSH_WAIT, function='_test_qemu_ssh')
+  def _test_qemu_ssh(self):
+    """ Tests that a VM is up and can be successfully SSH'd into.
```

Comment:
Why do you have this blank space at the beginning and end of every docstring? Can you get rid of it?